Microsoft.Toolkit.Uwp.UI.Controls/Loading/Loading.xaml

```diff
@@ -0,0 +1,112 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" 
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
+    
+    <Style TargetType="controls:Loading">
+        <Setter Property="HorizontalContentAlignment" Value="Stretch"/>
+        <Setter Property="VerticalContentAlignment" Value="Stretch"/>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:Loading">
+                    <Grid x:Name="RootGrid" HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
```

Comment:
Change to a Border control. Template bind background. Horizontal/VerticalAlignment should left alone. The border will expand to the values set by HorizontalAlignment and VerticalAlignment of the control itself.